git_big/__init__.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.6.4'
+__version__ = '0.6.3'
```

Comment:
did you mean to revert the version here?

Reply:
Yep. I tried to publish 0.6.3, ran into issues because my setuptools was too old. Then I thought I fixed it with 0.6.4, but was still broken. So, now that pypi lets me retract a publication, I'll just publish 0.6.3 like I originally planned.